.github/workflows/ci.yml

```diff
@@ -68,9 +68,9 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: "14.x"
-      - name: Yarn install
-        run: yarn
+          node-version: "18.x"
+      - name: NPM install
+        run: npm install
```

Comment:
Should we use `npm ci` here instead? It will install from `package-lock.json` instead of updating any dependency patches, etc.

Reply:
good idea 👍